tests/backup/backup_helper_test.go

```diff
@@ -0,0 +1,1205 @@
+				Name: orgID,
+			},
+		}
+		_, err := backupDriver.Delete(req)
+		Expect(err).NotTo(HaveOccurred())
+	})
+}*/
+
+func generateEncryptionKey() string {
+	var lower = []byte("abcdefghijklmnopqrstuvwxyz")
+	var upper = []byte("ABCDEFGHIJKLMNOPQRSTUVWXYZ")
+	var number = []byte("0123456789")
+	var special = []byte("~=+%^*/()[]{}/!@#$?|")
+	allChar := append(lower, upper...)
+	allChar = append(allChar, number...)
+	allChar = append(allChar, special...)
+
+	b := make([]byte, 12)
+	// select 1 upper, 1 lower, 1 number and 1 special
+	b[0] = lower[rand.Intn(len(lower))]
+	b[1] = upper[rand.Intn(len(upper))]
+	b[2] = number[rand.Intn(len(number))]
+	b[3] = special[rand.Intn(len(special))]
+	for i := 4; i < 12; i++ {
+		// randomly select 1 character from given charset
+		b[i] = allChar[rand.Intn(len(allChar))]
+	}
+
+	//shuffle character
+	rand.Shuffle(len(b), func(i, j int) {
+		b[i], b[j] = b[j], b[i]
+	})
+
+	return string(b)
+}
+
+func GetScheduleUID(scheduleName string, orgID string, ctx context.Context) (string, error) {
+	backupDriver := Inst().Backup
+	bkpUid, err := backupDriver.GetBackupUID(ctx, scheduleName, orgID)
+	if err != nil {
+		return "", fmt.Errorf("failed to get backup UID for %s, Err: %v", scheduleName, err)
+	}
+	backupSchInspectRequest := &api.BackupScheduleInspectRequest{
+		Name:  scheduleName,
+		Uid:   bkpUid,
+		OrgId: orgID,
+	}
+	resp, err := backupDriver.InspectBackupSchedule(ctx, backupSchInspectRequest)
+	if err != nil {
+		return "", fmt.Errorf("failed to inspect backup [%s] with UID [%s], Err: %v", scheduleName, bkpUid, err)
+	}
+	scheduleUid := resp.GetBackupSchedule().GetUid()
+	log.InfoD("Schedule Name - %s Schedule UID - %s", scheduleName, scheduleUid)
+
+	return scheduleUid, err
+}
+
+func removeStringItemFromSlice(itemList []string, item []string) []string {
+	sort.Sort(sort.StringSlice(itemList))
+	for _, element := range item {
+		index := sort.StringSlice(itemList).Search(element)
+		itemList = append(itemList[:index], itemList[index+1:]...)
+	}
+	return itemList
+}
+
+func removeIntItemFromSlice(itemList []int, item []int) []int {
+	sort.Sort(sort.IntSlice(itemList))
+	for _, element := range item {
+		index := sort.IntSlice(itemList).Search(element)
+		itemList = append(itemList[:index], itemList[index+1:]...)
+	}
+	return itemList
+}
+
+func getAllBackupLocations(ctx context.Context) (map[string]string, error) {
+	backupLocationMap := make(map[string]string, 0)
+	backupDriver := Inst().Backup
+	backupLocationEnumerateRequest := &api.BackupLocationEnumerateRequest{
+		OrgId: orgID,
+	}
+	response, err := backupDriver.EnumerateBackupLocation(ctx, backupLocationEnumerateRequest)
+	if err != nil {
+		return backupLocationMap, err
+	}
+	if len(response.BackupLocations) > 0 {
+		log.Infof("Found backup locations - ")
+		for _, backupLocation := range response.BackupLocations {
+			backupLocationMap[backupLocation.Uid] = backupLocation.Name
+			log.Infof(backupLocation.Name)
```

Comment:
add more to the log statement or remove it if it is added for debuging

Reply:
done